### PR TITLE
[MNT-23264] script execution error

### PR DESCRIPTION
**Pull request by code4uuuu**
https://hyland.atlassian.net/browse/MNT-23264

### CODE REVIEW

**Review by mohit-singh4 — APPROVED** [review]
LGTM

**Review by SathishK-T — APPROVED** [review]
LGTM

File: repository/src/main/java/org/alfresco/repo/action/executer/ScriptActionExecuter.java [hunk 1]
```diff
@@ -152,6 +156,11 @@ protected void executeImpl(Action action, NodeRef actionedUponNodeRef)
             {
                 // get the references we need to build the default scripting data-model
                 String userName = this.serviceRegistry.getAuthenticationService().getCurrentUserName();
+                if(SYSTEM_USER.equals(userName))
```
Comment (suneet-gupta) [inline repository/src/main/java/org/alfresco/repo/action/executer/ScriptActionExecuter.java:159]: check formatting

Comment (code4uuuu) [reply repository/src/main/java/org/alfresco/repo/action/executer/ScriptActionExecuter.java]: Resolved

**Comment by jottley** [comment]
This fix is not acceptable. Please see the notes added to the MNT and revert this change.

**Pull request merged**